src/index.js

```diff
@@ -117,12 +113,14 @@ export const getTargets = (targets = {}) => {
 
     delete targetOps.electron;
   }
+  browserslist.defaults = objectToBrowserslist(targetOps);
+
+  const browsersQuery = targetOps.browsers;
+  const browserslistOpts = {path: opts.path};
```

Comment:
Should we rename the `browserslist` path option on our end to something more descriptive?

Maybe `browserslistConfigPath`?